### PR TITLE
New send animation

### PR DESCRIPTION
**Pull request by cecileboucheron**
This changes the current animation to a circular loader. 
cc @adamjspooner can you try and see?

Encrypting:
image
Sent:
image
Error:
image

### CODE REVIEW

**Comment by adamjspooner** [comment]
@keybase/react-hackers, this is ready for review.

File: shared/chat/conversation/messages/wrapper/index.tsx [hunk 1]
```diff
@@ -841,16 +853,16 @@ const styles = Styles.styleSheetCreate(
       }),
       paddingLeftTiny: {paddingLeft: Styles.globalMargins.tiny},
       send: Styles.platformStyles({
-        common: {position: 'absolute'},
+        // common: {position: 'absolute'},
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/wrapper/index.tsx:856]: 🔥

File: shared/chat/conversation/messages/wrapper/index.tsx [hunk 2]
```diff
@@ -841,16 +853,16 @@ const styles = Styles.styleSheetCreate(
...
         isElectron: {
           pointerEvents: 'none',
-          right: 8,
-          top: 2,
-        },
-        isMobile: {
-          right: 0,
-          top: -8,
+          // right: 16,
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/wrapper/index.tsx:859]: 🔥

File: shared/chat/conversation/messages/wrapper/index.tsx [hunk 3]
```diff
@@ -841,16 +853,16 @@ const styles = Styles.styleSheetCreate(
...
+          // top: 3,
         },
+        // isMobile: {
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/wrapper/index.tsx:862]: 🔥

File: shared/chat/conversation/messages/wrapper/send-indicator/index.tsx [hunk 4]
```diff
@@ -116,17 +127,29 @@ class SendIndicator extends React.Component<Props, State> {
...
 
+  private animationType = () => {
+    let animationType = Styles.isDarkMode()
+      ? statusToIconDark[this.state.animationStatus]
+      : statusToIcon[this.state.animationStatus]
+    // There is no exploding-error state
+    if (this.props.isExploding && this.state.animationStatus !== 'error') {
+      // Don't show the sent state when exploding
+      if (this.state.animationStatus === 'sent') {
+        this.setState({visible: false})
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/wrapper/send-indicator/index.tsx:138]: we should not call setState in a render

Comment (adamjspooner) [reply shared/chat/conversation/messages/wrapper/send-indicator/index.tsx:138]: What's your recommendation?

Comment (adamjspooner) [reply shared/chat/conversation/messages/wrapper/send-indicator/index.tsx]: How's this? https://github.com/keybase/client/pull/23464/commits/7d22950232adb309b7a7dc787d136e5d22b503be

Comment (chrisnojima) [reply shared/chat/conversation/messages/wrapper/send-indicator/index.tsx]: sure

**Pull request merged**